src/routes/auth.js

```diff
@@ -18,7 +18,8 @@ const {
 
 const router = express.Router()
 const ERROR_MAP = {
-	[ refreshTokenIsInvalidError.name ] : 401
+	[ RefreshTokenIsInvalidError.name ] : 401,
+	[ InvalidInviteError.name ]  : 409
```

Comment:
they're basically unauthorised with an expired token so `401` is fine

Reply:
sure